container_images/bentoml-monitor-exporter/Makefile

```diff
@@ -1,5 +1,7 @@
-IMG ?= quay.io/bentoml/bentoml-monitor-exporter:0.0.2
+IMG ?= quay.io/bentoml/bentoml-monitor-exporter:0.0.3
 
 build:
 	docker build . --tag $(IMG)
+
+push:
```

Comment:
Follow the existing specification or change all Makefiles to the same specification. How about this?

https://github.com/bentoml/yatai-deployment/blob/main/container_images/bento-debugger/Makefile#L3

Reply:
sure